src/components/Homepage/HomePage.js

```diff
@@ -0,0 +1,15 @@
+import Banner from './Banner'
+import Feed from './Feed'
+import Article from './Article'
+
+
+function HomePage(){
+  return(
+    <div>
+      <Banner/>
+      <Feed/>
+      <Article/>
```

Comment:
Article should not be here, it should instead be called in Feed. (it should still be its own component.)

Reply:
To clarify, Should I put the "Article" component and the "Popular Tags" component into the "Feed" component?